lib/phoenix/live_dashboard/pages/ecto_stats_page.ex

```diff
@@ -170,4 +258,25 @@ defmodule Phoenix.LiveDashboard.EctoStatsPage do
 
   defp format(_type, value),
     do: value
+
+  defp render_error(assigns) do
+    error_message =
+      case assigns.error do
+        :repos_auto_discovery_not_available ->
+          "The version of Ecto installed does not support the auto discovery of repos. Please update your Ecto to >= 3.6.2"
```

Comment:
I don't think this is necessary now because we are requiring 3.6.2, right?

Reply:
Yeah, I added emulating the Broadway dashboard, because the remote node may not have it. But I will handle this better when changing to multi nodes/plds.